### DISCUSSION TITLE
Welcome to deimos Discussions!

### DISCUSSION THREAD

**Discussion by dorner** [flipp-oss/deimos#145] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iMacTia** [comment]
Hey everyone 👋, I'm Matt and I recently started using `deimos` in my company as we migrate towards asynchronous communication between services and implement a "data mesh" approach in our architecture.
This seemed the most complete framework, so we'd be happy to contribute back whenever we need to add more features or fix any bugs 😄

**Comment by dorner** [comment]
Welcome! You're the second company I'm aware of using the framework so we're super happy to have more eyeballs on it!
